plugins/catalog-react/src/components/CatalogFilterLayout/CatalogFilterLayout.tsx

```diff
@@ -28,14 +28,22 @@ import FilterListIcon from '@material-ui/icons/FilterList';
 import { BackstageTheme } from '@backstage/theme';
 
 /** @public */
-export const Filters = (props: { children: React.ReactNode }) => {
-  const isMidSizeScreen = useMediaQuery<BackstageTheme>(theme =>
-    theme.breakpoints.down('md'),
+export type FiltersOptions = {
```

Comment:
i started thinking, should this one be named `CatalogFilterLayoutFiltersOptions` to be properly scoped, but that felt a bit awkward so ... ended up thinking that maybe it doesn't need a named type at all and can be inlined instead :) Does that sound sensible? Otherwise I vote `CatalogFilterLayoutFiltersOptions`.